test/suites/storage_buckets.sh

```diff
@@ -194,3 +194,91 @@ test_storage_buckets() {
     deconfigure_loop_device "${loop_file_1}" "${loop_device_1}"
   fi
 }
+
```

Comment:
You're defining a new test but not adding it to `main.sh`, so it's currently not being run.